terraform/groups/ecs-service/profiles/development-eu-west-2/phoenix1/vars

```diff
@@ -2,8 +2,7 @@ environment = "phoenix1"
 aws_profile = "development-eu-west-2"
 
 # service configs
-use_set_environment_files = true
-log_level = "debug"
+log_level = "info"
 
 # Scheduled scaling of tasks
 service_autoscale_enabled  = true
```

Comment:
```suggestion
```
Im suspecting this one isnt needed too, as it is true by default in variables.tf